lib/form1010_ezr/service.rb

```diff
@@ -72,11 +72,6 @@ def log_submission_failure(parsed_form)
       StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.failed")
 
       if parsed_form.present?
-        PersonalInformationLog.create!(
-          data: parsed_form,
-          error_class: 'Form1010Ezr Failed'
-        )
-
```

Comment:
By removing this, we'll only create a `PersonalInformationLog` on validation failures and total failures.